forge/forge/module.py

```diff
@@ -963,6 +963,12 @@ def _set_device(self, device: "Device"):
         for submodule in self.get_submodules().values():
             submodule._set_device(device)
 
+    def process_framework_parameters(self):
```

Comment:
What is the point of this one? I am not that familiar with unique op test generation, so maybe I am missing something...

Reply:
Great, thanks!